roles/varnish/templates/default.vcl.j2

```diff
@@ -74,7 +74,7 @@ sub vcl_recv {
 {%if proxy_cache_block_basic_auth|default(True) %}
     # Don't allow basic auth via this route.
     # If you want to login at the Zope root, use an ssh tunnel.
-    if (req.http.Authorization) {
+    if (req.http.Authorization && req.http.Authorization !~ 'Bearer') {
```

Comment:
What is the `!~` operator? That's a new one to me.

Reply:
https://varnish-cache.org/docs/2.1/tutorial/vcl.html#operators

`~` is a regex match and `!~` negates it, so it means "does not match".

Reply:
Note that both `~` and `!~` are used elsewhere in the same vcl templates.

Reply:
Oh, derp! I thought it was Python, not Varnish config.